torchvision/models/quantization/mobilenetv3.py

```diff
@@ -135,20 +136,30 @@ def _mobilenet_v3_model(
         if "backend" in weights.meta:
             _ovewrite_named_param(kwargs, "backend", weights.meta["backend"])
     backend = kwargs.pop("backend", "qnnpack")
+    fx_mode = kwargs.pop("fx_mode", False)
```

Comment:
same here, can this flag be named in a way that's specific to quantization?  The official name of the workflow is "FX graph mode quantization".

Reply:
Ok, I think I will rename both flags to the same thing `quantization_workflow_type` and just check if it's equal to `"fx"`

Reply:
nit: it's still not ideal to use "fx" as a string value, the workflow name is "FX graph mode quantization", it would be good to be consistent to minimize confusion.  There are also other FX passes people might be using.